test/unittests/libweb3jsonrpc/jsonrpc.cpp

```diff
@@ -319,6 +336,90 @@ BOOST_AUTO_TEST_CASE(eth_sendTransaction)
     BOOST_CHECK_EQUAL(txAmount, balance2);
 }
 
+BOOST_AUTO_TEST_CASE(eth_sendRawTransaction)
+{
+    auto senderAddress = coinbase.address();
+    auto receiver = KeyPair::create();
+    
+
+    BOOST_TEST_CHECKPOINT("Sending a signed transaction from 0 balance account should fail");
+
+    int blockNumber = 0;
+    BOOST_CHECK_EQUAL(0, web3->ethereum()->balanceAt(senderAddress, blockNumber));
+
+    Json::Value t;
+    t["from"] = toJS(senderAddress);
+    t["to"] = toJS(receiver.address());
+    t["value"] = jsToDecimal(toJS(0.01));

```

Comment:
I think value is always in wei (so should be integer) and this will result in transaction sending 0 value